### ISSUE TITLE
[📖 Docs]: Link accessibility issue on homepage

### ISSUE THREAD

**Issue by StephNathai** [webdriverio/webdriverio#14690]
### Pre-check

- [x] I know I can edit the docs but prefer to file this issue

### Describe the improvement

Other

### Description of the improvement / report

Noticed an AxeDevTools accessibility callout on the homepage where the React testing web component link looks to be missing an alt tag, making it inaccessible. All links should have discernible text as stated [here](https://dequeuniversity.com/rules/axe/4.10/link-name?application=AxeChrome) by Deque University.

<img width="1888" height="1014" alt="Image" src="https://github.com/user-attachments/assets/d936f506-a0f4-4c56-abdc-8639a618afe8" />

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by StephNathai** [comment]
I will create a PR to fix this issue shortly.

**Comment by christian-bromann** [comment]
> I will create a PR to fix this issue shortly.

That would be much appreciated 🙏

**Comment by Yogavenkatanarayanan** [comment]
@StephNathai I would like to address this issue if it is not already done by you yet. Let me know!
